docker-compose.yaml

```diff
@@ -233,19 +233,19 @@ services:
       - "8181:8181"
 
   openldap:
-    image: bitnami/openldap:2.6.6
+    image: bitnami/openldap:2.6.7
     ports:
       - "1389:1389"
       - "1636:1636"
     volumes:
       - ./scripts/docker-compose/imports/openldap/schemas:/schemas
-      - ./scripts/docker-compose/imports/openldap/ldifs:/ldifs
       - ./scripts/docker-compose/imports/openldap/zac-scripts:/zac-scripts
     environment:
       - LDAP_ADMIN_USERNAME=admin
       - LDAP_ADMIN_PASSWORD=admin
       - LDAP_ROOT=dc=example,dc=org
       - LDAP_SKIP_DEFAULT_TREE=yes
+      - LDAP_CUSTOM_LDIF_DIR=/tmp
```

Comment:
Ah nice! I also looked at this option at some point. Not too sure why I did not use it back then but oh well.